### ISSUE TITLE
Can I use this repo for models Tiny models created in Darknet?

### ISSUE THREAD

**Issue by JakupGuven** [qqwweee/keras-yolo3#407]
I'm training custom models using the Darknet framework using the YOLOv2 and YOLOv3 tiny architectures. When I convert these models to Keras format with your repo @qqwweee and subsequently use coremltools (made by Apple to convert keras and other formats to Apple formats) it doesn't work. I've managed to convert YOLO keras models to CoreML before using YAD2K.

**Comment by TeMob** [comment]
I'd like to know the same. I've used this repo to convert from tiny-yolov3 to keras to coreML in the past, which worked well. But with my newest project I can't get it to work either. The model in Darknet detects everything, but after the conversion, it isn't able to detect a single object.

**Comment by JakupGuven** [comment]
@TeMob are you using Darknet by AlexeyAB? I found that they had switched from saving weights as 32-bit unsigned integers to 64 bit ones which messed up the CoreML model, by changing it back I solved it.

**Comment by TeMob** [comment]
@JakupGuven Yes I am. That's interesting! Where would I go change this? Thanks

**Comment by JakupGuven** [comment]
@TeMob
You'll find everything here:
https://github.com/AlexeyAB/darknet/issues/2922#issuecomment-482811870
